### PR TITLE
Port PyHeufyBot's time command

### PR DESCRIPTION
**Pull request by Heufneutje**
Wanted to replace our current time module but since both function differently I made my its own command.

### CODE REVIEW

**Comment by HubbeKing** [comment]
Why not simply remove the "time" trigger stuff from the existing Time.py module, rename the file to Date.py, and add this module/command as Time.py/time? Nothing is actually using DesertBot's existing .time command, so...

File: desertbot/modules/commands/TimeZone.py [hunk 1]
```diff
@@ -0,0 +1,129 @@
...
+        properDay = self._getProperDay(resultDate.day)
+        formattedTime = resultDate.strftime("%H:%M (%I:%M %p) on %A, " + properDay + " of %B, %Y")
+        return "Timezone: {} | Local time is {}".format(timeJSON["timeZoneName"], formattedTime)
+
+    def _getProperDay(self, day):
+        if day in [1, 21, 31]:
+            return "{}st".format(day)
+        elif day in [2, 22]:
+            return "{}nd".format(day)
+        elif day in [3, 33]:
```
Comment (HubbeKing) [inline desertbot/modules/commands/TimeZone.py]: 23rd also

**Comment by Heufneutje** [comment]
Travis error seems unrelated to what I changed.

**Review by HubbeKing — APPROVED** [review]
Looks good to me.

If we really want to, we can later figure out some way to recreate the old 'time' command, maybe even just a new trigger word for it.

**Pull request merged**